src/GitHub/Data/GitData.hs

```diff
@@ -184,9 +184,9 @@ instance NFData GitUser where rnf = genericRnf
 instance Binary GitUser
 
 data File = File
-    { fileBlobUrl   :: !URL
+    { fileBlobUrl   :: !(Maybe URL)
```

Comment:
won't we be on safer ground if `FromJSON` instance is changed to use `:?` for changed fields?

Otherwise LGTM

Reply:
Good catch. Added in b4422b019605d6e60e5f4b612413bd0e79fc3a59